addons/mail/static/src/models/rtc.js

```diff
@@ -759,12 +755,12 @@ registerModel({
                 dataChannel.close();
             }
             delete this._dataChannels[sessionId];
-            const peerConnection = this._peerConnections[sessionId];
-            if (peerConnection) {
-                this._removeRemoteTracks(peerConnection);
```

Comment:
I think that everything in `_removePeer()` can be moved to inside `RtcSession.reset()` (which could be called `RtcSession.disconnect()`, after https://github.com/odoo/odoo/pull/85024 and https://github.com/odoo/odoo/pull/85032 are merged.

Reply:
Good idea! Will do this as finishing cleaning of turning all `Rtc` attributes into `RtcSession` fields.